### PR TITLE
Adding startup scripts for the analytics profile

### PR DESCRIPTION
**Pull request by dilee**
## Purpose
Startup scripts for the WSO2 Stream Processor analytics profile.

## Goals
N/A

## Approach
N/A

## User stories
N/A

## Release note
N/A

## Documentation
N/A

## Training
N/A

## Certification
N/A

## Marketing
N/A

## Automation tests
N/A

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? N/A
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? no

## Samples
N/A

## Related PRs
N/A

## Migrations (if applicable)
N/A

## Test environment
N/A
 
## Learning
N/A

### CODE REVIEW

File: distribution/src/scripts/analytics-dashboard.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+case "`uname`" in
+CYGWIN*) cygwin=true;;
+MINGW*) mingw=true;;
+OS400*) os400=true;;
+Darwin*) darwin=true
+        if [ -z "$JAVA_VERSION" ] ; then
+             JAVA_VERSION="CurrentJDK"
+           else
+             echo "Using Java version: $JAVA_VERSION"
+           fi
```
Comment (prabushi) [inline distribution/src/scripts/analytics-dashboard.sh]: Fix the formatting issue.

File: distribution/src/scripts/analytics-worker.sh [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+case "`uname`" in
+CYGWIN*) cygwin=true;;
+MINGW*) mingw=true;;
+OS400*) os400=true;;
+Darwin*) darwin=true
+        if [ -z "$JAVA_VERSION" ] ; then
+             JAVA_VERSION="CurrentJDK"
+           else
+             echo "Using Java version: $JAVA_VERSION"
+           fi
```
Comment (prabushi) [inline distribution/src/scripts/analytics-worker.sh]: Fix the formatting issue.

File: distribution/src/scripts/analytics-dashboard.sh (hunk 1 above)
Comment (dilee) [reply distribution/src/scripts/analytics-dashboard.sh]: Fixed in https://github.com/wso2/product-ei/pull/3665

File: distribution/src/scripts/analytics-worker.sh (hunk 2 above)
Comment (dilee) [reply distribution/src/scripts/analytics-worker.sh]: Fixed in https://github.com/wso2/product-ei/pull/3665

**Pull request merged**